### PR TITLE
Document pitfalls of writing policy rules

### PR DESCRIPTION
**Pull request by lcarva**
Ref: EC-517

### CODE REVIEW

**Review by zregvart — APPROVED** [review]
Skimmed through, LGTM

**Comment by lcarva** [comment]
Let's get this in. This is basically a copy of an internal document that most of us has already reviewed anyways. As always, post-merge reviews are always welcome.

File: antora/docs/modules/ROOT/pages/authoring.adoc [hunk 1]
```diff
@@ -58,3 +58,88 @@ further reference on annotations.
...
+	count(attestations) == 0
+	result := {"msg": "No attestation found"}
+}
+
+attestations := object.get(input, "attestations", [])
+
+statement_type(att) := object.get(att, ["statement", "_type"], "N/A")
+
+predicate_type(att) := object.get(att, ["statement", "predicateType"], "N/A")
+```
```
Comment (simonbaird) [inline antora/docs/modules/ROOT/pages/authoring.adoc:145]: Fantastic!

This could make for a nice blog post probably.

**Comment by simonbaird** [comment]
I'm too slow, but lgtm, no suggestions.

**Pull request merged**